dev/src/DocFx/Toc/ClassToc.php

```diff
@@ -42,6 +42,18 @@ public function isServiceClass(): bool
         return $this->classNode->isServiceClass();
     }
 
+    public function isRequestClass(): bool
+    {
+        return $this->isProtobufMessageClass()
+            && 'Request' === substr($this->classNode->getName(), -7);
```

Comment:
Just wanted to verify this heuristic doesn't have any odd edge cases, have you checked?

Reply:
I found exceptions where a protobuf message ends in `Request` but isn't a true RPC request (for example, `Google\Cloud\VideoStitcher\V1\AdRequest`). This happens in about 12 components.

I also found exceptions where a protobuf message does NOT end in `Request, but _is_ a true RPC request (for example, `Google\Cloud\AccessApproval\V1\ApproveApprovalRequestMessage`)

But until some action is taken in protobuf or in the GAPIC generator, there's no way for Cloud RAD to know for sure which of these are requests or responses.

Reply:
I will try to use reflection in the GAPIC clients to compile a list of requests, and see if that works.